src/ripple/app/misc/impl/ValidatorList.cpp

```diff
@@ -165,20 +174,22 @@ ValidatorList::load (
 
         if (!id)
         {
-            JLOG (j_.error()) <<
-                "Invalid node identity: " << match[1];
+            JLOG (j_.error()) << "Invalid node identity: " << match[1];
             return false;
         }
 
-        if (trusted (*id))
+        // Skip local key which was already added
+        if (*id == localPubKey_ || *id == localSigningKey)
+            continue;
+
+        auto ret = keyListings_.insert ({*id, 1});
+        if (! ret.second)
         {
-            JLOG (j_.warn()) <<
-                "Duplicate node identity: " << match[1];
+            JLOG (j_.warn()) << "Duplicate node identity: " << match[1];
             continue;
         }
-
-        if (insertPermanentKey(*id, trim_whitespace (match[2])))
-            ++count;
+        publisherLists_[local].list.emplace_back (std::move(*id));
```

Comment:
That was the intention. I basically need a dummy entry in `publisherLists` representing validators listed in the config (if there are any). I think it is only necessary when checking if there is a single list here:
https://github.com/ripple/rippled/pull/1842/commits/00b5014a9b51e34a23c2560c61241e01bf566543#diff-bd1a4f40bdb7b8c2b62762e9078a8fb2R377
which probably means I don't actually need to populate the list.